docs/configurations.md

```diff
@@ -37,6 +37,7 @@ var integrationSdk = sharetribeIntegrationSdk.createInstance({
   // HTTP and HTTPS agents to be used when performing
   // http and https request. This allows defining non-default
   // options for agent, such as `{ keepAlive: false }`.
+  // When creating custom httpsAgent, set `maxSockets` value to 10 or less.
```

Comment:
This could be validated in the constructor, that if a custom agent is passed, `maxSockets` is 10 or less.

Reply:
Hm, throwing an error would be a breaking change in the SDK, but perhaps a warning could be printed out?

Reply:
True that, a warning likely is fine. Especially, as with the default http agent, there's no limit to concurrent requests. Therefore, allowing higher than 10 there is not a problem.

Reply:
Added a warning!